### PR TITLE
Replacing ADES with ACON in ramp

### PR DESCRIPTION
**Pull request by lisazacarias**
- replacing ADES spinbox with ACON label
- ramping to ACON instead of stored ADES value
- reformatting

### CODE REVIEW

File: setup_gui.py [hunk 1]
```diff
@@ -3,45 +3,61 @@
...
-                                                       WorkerSignals)
+from lcls_tools.common.pydm_tools.displayUtils import (
+    ERROR_STYLESHEET,
+    STATUS_STYLESHEET,
+    WorkerSignals,
+)
 from lcls_tools.common.pyepics_tools.pyepics_utils import PV, PVInvalidError
 from lcls_tools.superconducting import sc_linac_utils
-from lcls_tools.superconducting.scLinac import (CRYOMODULE_OBJECTS, Cavity)
+from lcls_tools.superconducting.scLinac import CRYOMODULE_OBJECTS, Cavity
```
Comment (nneveu) [inline setup_gui.py:27]: Are the brackets only needed for multiple lines?

File: setup_gui.py [hunk 2]
```diff
@@ -342,102 +389,116 @@ def add_cm_tab(self, cm_name: str):
...
+
+            cav_vlayout.addLayout(cav_amp_hlayout)
             cav_vlayout.addLayout(cav_button_hlayout)
             cav_vlayout.addWidget(cav_widgets.status_label)
-            all_cav_layout.addWidget(cav_groupbox,
-                                     0 if cav_num in range(1, 5) else 1,
-                                     (cav_num - 1) % 4)
+            all_cav_layout.addWidget(
+                cav_groupbox, 0 if cav_num in range(1, 5) else 1, (cav_num - 1) % 4
+            )
```
Comment (nneveu) [inline setup_gui.py:427]: What's cav_groupbox? CM# and Cavity#? Why only 0 or 1? Why cav_num -1?

File: setup_gui.py [hunk 3]
```diff
@@ -342,102 +389,116 @@ def add_cm_tab(self, cm_name: str):
...
+                    settings=self.settings,
+                    parent=self,
+                )
+            )
+
+        self.linac_widgets.insert(
+            2,
+            Linac(
+                "L1BHL", 1, sc_linac_utils.L1BHL, settings=self.settings, parent=self
+            ),
```
Comment (nneveu) [inline setup_gui.py:468]: Why a ',' here?

**Review by nneveu — APPROVED** [review]
I asked a few comments for my learning/awareness, but I think it looks pretty good.

File: setup_gui.py (hunk 1 above)
Comment (lisazacarias) [reply setup_gui.py:27]: Yes!

File: setup_gui.py (hunk 2 above)
Comment (lisazacarias) [reply setup_gui.py:427]: can_groupbox is the QGroupBox that holds all of a cavity's widgets (the labels and buttons and things).
The arguments for all_cav_layout are (widget, row number, column number), which are 0 indexed (whereas cavity numbering is 1 indexed, hence the -1). I only have two rows, so row 0 for rack A and row 1 for rack B.

File: setup_gui.py (hunk 3 above)
Comment (lisazacarias) [reply setup_gui.py:468]: No idea, Black put it there lol Apparently Black refuses to let me remove it

**Pull request merged**